staff/henry-novoa/cv/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Page Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
```

Comment:
In this case you don't have any css stylesheet, right??

Reply:
Not in this case, I used the standard template available in VCS

Reply:
remove CSS link then, it is NOT required for the challenge